cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/deploy/pipeline/CreateSystemTablesStage.java

```diff
@@ -46,6 +46,7 @@ public CreateSystemTablesStage(StructuredTableAdmin structuredTableAdmin) {
   public void process(ApplicationDeployable input) throws IOException, TableAlreadyExistsException {
     for (StructuredTableSpecification spec : input.getSystemTables()) {
       try {
+        structuredTableAdmin.update(spec);
```

Comment:
Can we merge the create and update calls? Ideally it's an idempotent operation to make the StructuredTable schema to a desired state.

Reply:
updated the create method to createOrUpdate